grapher/barCharts/DiscreteBarChart.tsx

```diff
@@ -456,26 +456,33 @@ export class DiscreteBarChart
         return sortBy(raw, (series) => series.row.value)
     }
 
-    @computed private get valuesToColorsMap() {
-        const { manager } = this
-        const colorScheme = manager.baseColorScheme
-            ? ColorSchemes[manager.baseColorScheme]
-            : undefined
-        if (!colorScheme) return undefined
-
-        return colorScheme.getUniqValueColorMap(
-            uniq(this.sortedRawSeries.map((series) => series.row.value)),
-            manager.invertColorScheme
+    @computed private get colorScheme() {
+        // If this DiscreteBarChart stems from a LineChart, we want to match its (default) color
+        // scheme OWID Distinct. Otherwise, use an all-blue color scheme (`null`) as default.
+        const defaultColorScheme = this.manager.isLineChart
+            ? ColorSchemes["owid-distinct"]
+            : null
+
+        return (
+            (this.manager.baseColorScheme
+                ? ColorSchemes[this.manager.baseColorScheme]
+                : null) ?? defaultColorScheme
```

Comment:
Why `null` instead of `undefined`?

Reply:
No real reason, it just was like this before :)
But I can quickly change it, I also think `undefined` makes more sense.